src/Microsoft.VisualStudio.Threading/DispatcherExtensions.cs

```diff
@@ -0,0 +1,78 @@
+﻿/********************************************************
+*                                                        *
+*   © Copyright (C) Microsoft. All rights reserved.      *
+*                                                        *
+*********************************************************/
+
+#if NET45
```

Comment:
Our netstandard build is a subset of our net45 build. And yes, they share an assembly name. So if you compile against netstandard you'll work against the net45 version. But if you compile against the net45 version and run against the netstandard one you may experience missing types/members. That's the pattern we're already following.